pkgs/unified_analytics/lib/src/event.dart

```diff
@@ -357,6 +357,92 @@ final class Event {
           if (exitCode != null) 'exitCode': exitCode,
         };
 
+  /// Event that is sent from devtools for various different actions as
+  /// indicated by the [eventCategory].
+  Event.devtoolsAction({
+    required String eventCategory,
+    required String label,
+    required int value,
+    required bool nonInteraction,
+    required String userApp,
+    required String userBuild,
+    required String userPlatform,
+    required String devtoolsPlatform,
+    required String devtoolsChrome,
+    required String devtoolsVersion,
+    required String ideLaunched,
+    required String isExternalBuild,
+    required String isEmbedded,
+    required String ideLaunchedFeature,
```

Comment:
Can I have these values as required? Seems like they are not conditionally being added based on the screen metrics object used